docs/docs/mlflow-3/index.mdx

```diff
@@ -14,7 +14,7 @@ Discover the next generation of MLflow, designed to streamline your AI experimen
 ## What is MLflow 3.0?
 
 The best AI experimentation platform for building high quality models and GenAI agents! MLflow 3.0 introduces a variety of exciting improvements to MLflow Tracking and Tracing, including:
-- First class model support in MLflow tracking with **LoggedModel**
+- First class model support in MLflow tracking with <APILink fn="mlflow.entities.LoggedModel">**LoggedModel**</APILink>
```

Comment:
Can we describe what users can do / achieve with the feature, instead? "First class support" isn't super clear

Something like:

```
Centrally track and analyze the performance of your models and generative AI applications across all environments, from interactive queries in a development notebook through production batch or real-time serving deployments.
```

Reply:
Updated!